user-integration-guide/symfony-integration.md

```diff
@@ -0,0 +1,218 @@
+# Integrating Opentelemetry PHP into Symfony Applications
+
+## Introduction 
+
+As a developer, you might be wondering how  OpenTelemetry could be beneficial to you. You are not alone.  Without practical examples, the usefulness of distributed tracing can be difficult to grasp for persons without a cloud or site reliability engineering background. This user guide shows how OpenTelemetry could be useful to gain insights into exceptions happening within an application. This example uses the OpenTelemtry PHP library integrated into a Symfony application, bundled with Jaeger and Zipkin, for visualizing data.  
+## Prerequisites
+To follow this guide you will need:
+
+* PHP Installed, this example uses PHP 7.4 .
+* [Composer](https://getcomposer.org/download/ ) for dependency management. 
+* [Symfony CLI](https://symfony.com/download) for managing your Symfony application
+*  [Docker](https://docs.docker.com/get-docker/) for bundling our visualization tools
+
+## Step 1 - Creating a Symfony Application 
+
+Create a Symfony application by running the command `symfony new my_project_name`. Remember that you need the  [Symfony CLI](https://symfony.com/download) installed for the `symfony` command to work. I am calling this example `otel-php-symfony-basic-example`, so the command is as follows; `symfony new otel-php-symfony-basic-example` .
+
+## Step 2 - Require and Test Symfony Dependencies
+
+ To define our routes within our controller methods, let's require the Doctrine annotation library by running the command `composer require doctrine/annotations`.
+
+We can test that routes defined within Controllers work by creating a `HelloController.php` file within the `src\Controller` folder as follows:
+```
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class HelloController extends AbstractController
+{
+    /**
+     * @Route("/hello", name="hello")
+     */
+    public function index(): Response
+    { 
+        return new Response('Hello World');
+    }
+}
+```
+
+To check out the routes available in our current project run `php bin/console debug:router`. 
```

Comment:
You may want to add a screenshot of this (and perhaps some of the other actions you're taking in this document)?